src/www/css/main.css

```diff
@@ -46,4 +46,17 @@ pre#buildLog{
 
 #btnLive, #btnDownload, #deploy {
   margin-top : 1%;
+}
+
+div.form-div-board {
+	margin-bottom: 0;
+	padding: 20px;
+	background: #0000001a;
+	box-shadow: 5px 5px 5px rgba(0, 0, 0, 0.54);
+	margin-top: 40px;
+	margin-bottom: 50px;
+}
+
+div.form-div-board li{
+	color: #101012;
```

Comment:
@duttaditya18 Please do not change the color of text.

Reply:
Then tell me what should be the color of the background.